src/test/clojure/clara/test_rules.clj

```diff
@@ -1281,9 +1284,9 @@
                                           [:or
                                            [Cold (= ?temp temperature)
                                             (< temperature 10)]
-                                           [Windspeed (= ?loc location)
+                                           [WindSpeed (= ?loc location)
```

Comment:
This test was broken then I guess?

Reply:
Yes, the linter caught that this was broken, the rule in question supposed to fail anyways because it is for `test-unbound-bindings` but the binding was supposed to be wrong, not also the fact type.